contrib/sirupsen/logrus/example_test.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package logrus
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func ExampleHook() {
+	//Setup logrus, do this once at the beginning of your program
+	logrus.SetFormatter(&logrus.JSONFormatter{})
+	logrus.AddHook(&DDContextLogHook{})
+
+	span, sctx := tracer.StartSpanFromContext(context.Background(), "mySpan")
+
+	//Pass the current context to the logger
+	cLog := logrus.WithContext(sctx)
+	//Log as desired using the context-aware logger
+	cLog.Info("Completed some work!")
```

Comment:
Can you use `// Output: ` here? (https://go.dev/blog/examples)

Reply:
Ahh neat! I've never used `// Output: ` before. 

It seems to make the example a bit more clunky as it means that the output has to be set to stdout and the time overridden to a fixed value. I've pushed up the change with the `Output` validation, let me know if you think it's not worth the extra couple lines of complexity. I'm leaning towards it's not worth it given there's a unit test that validates the `Hook` method already and my hope for the example was it's mostly copy/paste for users.

Reply:
I think this is great actually. But frankly I was expecting the comment to be just below the log call that causes it. But perhaps I don't fully understand how `Output:` works. Disclaimer: I've never used it.